util/cron/nightly

```diff
@@ -697,9 +697,6 @@ if ($nolocal == 1) {
 if ($verify == 1) {
     $testflags = "$testflags -compopts --verify";
 }
-if ($llvm == 1) {
-    $testflags = "$testflags -compopts --llvm";
-}
```

Comment:
When this is merged, is there still a way to run the tests using the C back-end?

Reply:
`export CHPL_LLVM_BY_DEFAULT=0` or `-compopts --no-llvm` should both do that.

Reply:
> When this is merged, is there still a way to run the tests using the C back-end?

I think I misunderstood what you were implying and made a new commit to address it. The new commit adds a new `-no-llvm` flag to util/cron/nightly that will pass `-compopts --no-llvm` to the testflags.